bin/bulk_cli.py

```diff
@@ -0,0 +1,302 @@
+
+    if not vpc_config["Subnets"] and not vpc_config["SecurityGroupIds"]:
+        vpc_config = get_vpc_details(aws_region)
+
+        if vpc_config is None:
+            print("Failed to retrieve VPC details.")
+            sys.exit(1)
+
+    s3_bulk = S3_Bulk_Manager(input_bucket, aws_region, cluster_config["InstanceCount"])
+    s3_bulk.count_objects()
+    s3_bulk.create_and_upload_manifest_files()
+
+    sagemaker_client = boto3.client("sagemaker", aws_region)
+
+    print("Now submitting the jobs...")
+
+    for manifest_file in s3_bulk.manifest_filename:
+        s3_input_config["S3Uri"] = manifest_file
+
+        input_config = {
+            "InputName": "input",
+            "S3Input": s3_input_config,
+        }
+
+        output_config = {
+            "OutputName": "output",
+            "S3Output": s3_output_config,
+        }
+
+        cluster_config["InstanceCount"] = 1
+        job_name = f"bulk-ingest-job-{token_hex(16)}"
+        response_exec = sagemaker_client.create_processing_job(
+            ProcessingInputs=[input_config],
+            ProcessingOutputConfig={"Outputs": [output_config]},
+            ProcessingJobName=job_name,
+            ProcessingResources={"ClusterConfig": cluster_config},
+            StoppingCondition={"MaxRuntimeInSeconds": max_runtime},
+            AppSpecification={
+                "ImageUri": image_uri,
+                "ContainerEntrypoint": [
+                    "/entry.sh",
+                    "/bin/bash",
+                    "-c",
+                    "python3 osml-data-intake/src/aws/osml/data_intake/bulk_ingest/process_image.py",
+                ],
+            },
+            NetworkConfig={"EnableNetworkIsolation": False, "VpcConfig": vpc_config},
+            RoleArn=role_arn,
+            Environment={
+                "S3_URI": manifest_file,
+                "S3_INPUT_PATH": s3_input_config["LocalPath"],
+                "S3_OUTPUT_PATH": s3_output_config["LocalPath"],
+                "S3_OUTPUT_BUCKET": output_bucket,
+                "STAC_ENDPOINT": stac_endpoint,
+                "COLLECTION_ID": collection_id,
+                "ES_HOST": opensearch_config["ES_HOST"],
+                "ES_PORT": opensearch_config["ES_PORT"],
+                "ES_USE_SSL": opensearch_config["ES_USE_SSL"],
+                "ES_VERIFY_CERTS": opensearch_config["ES_VERIFY_CERTS"],
+                "THREAD_WORKERS": str(thread_workers),
+            },
+        )
+
+        print(f"{job_name} has been submitted!")
+
+        if "ProcessingJobArn" in response_exec:
+            arn = response_exec["ProcessingJobArn"]
+            job_name = arn.split("/")[-1]
+            status_code = response_exec["ResponseMetadata"]["HTTPStatusCode"]
+
+            print(f"Job Name: {job_name:<15}\nStatus Code:{status_code:<15}\nArn: {arn:<50}\n")
+        else:
+            print("Error: Failed to retrieve processing job details.\n")
+
+
+if __name__ == "__main__":
+    execute_sm_processing_job()
```

Comment:
May want to switch to use the logger instead of print.  That would also give the ability for a --verbose argument tied to the log level for more detailed output